index.js

```diff
@@ -43,17 +38,32 @@ module.exports = (robot, options, visit) => {
     });
   }
 
+  // Trigger the event
+  function trigger(event) {
+    robot.webhook.emit(event.event, event);
+  }
+
   function schedule(installation, repository) {
```

Comment:
should an additional variable be supported so you could have multiple scheduled events within a single probot app (especially if you are combining different functionality)? you could then pass this variable/name and append it onto the event name to provide uniqueness.

aka, `event-hourly` vs `event-daily` vs `event-per-minute`, etc.